scipy/stats/tests/test_distributions.py

```diff
@@ -2715,6 +2726,41 @@ def test_edge_cases(self):
             assert_equal(stats.triang.cdf(1., 1.), 1)
 
 
+class TestBurr(object):
+    def test_endpoints_7491(self):
+        # gh-7491
+        # Compute the pdf at the left endpoint dst.a.
```

Comment:
There were also some changes to `_cdf` and logged versions of these methods too. Those should be tested as well in this `TestBurr` class.

Reply:
Those changes are tested in other generic routines, now that the list of of x-values to be tested has bene expanded to include (finite) endpoints.

Reply:
right, the generic tests for all distributions will get test the changes for pdf, cdf and log versions.

Reply:
ok, that makes sense.